src/tests/unit/tests/DetailsView/components/left-nav/assessment-left-nav.test.tsx

```diff
@@ -20,7 +20,7 @@ import { LeftNavLinkBuilder } from '../../../../../../DetailsView/components/lef
 import { NavLinkHandler } from '../../../../../../DetailsView/components/left-nav/nav-link-handler';
 import { DictionaryStringTo } from '../../../../../../types/common-types';
 
-describe('AssessmentLeftNav', () => {
+describe(AssessmentLeftNav, () => {
```

Comment:
This looks like it's missing the single quotes

Reply:
We can directly put the class name/function name here instead of the string.

Reply:
It takes: name: string | number | Function | jest.FunctionLike, in our case I find put the Function itself would be the best.